warehouse/accumulo-extensions/src/main/java/datawave/ingest/table/volumeChoosers/ShardedTableDateBasedTieredVolumeChooser.java

```diff
@@ -0,0 +1,67 @@
+package datawave.ingest.table.volumeChoosers;
+
+import org.apache.accumulo.server.fs.RandomVolumeChooser;
+import org.apache.accumulo.server.fs.VolumeChooserEnvironment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ShardedTableDateBasedTieredVolumeChooser extends RandomVolumeChooser {
+    private static final Logger log = LoggerFactory.getLogger(ShardedTableDateBasedTieredVolumeChooser.class);
+    private static final String NEW_SUFFIX = "volume.dates.tiered.new";
+    private static final String OLD_SUFFIX = "volume.dates.tiered.old";
+    private static final String DAYS_BACK_SUFFIX = "volume.dates.tiered.days.back";
+    private static final String DATE_PATTERN = "yyyyMMdd";
+    
```

Comment:
I seems to me this could simply be extended to handle more than 2 tiers.  It might be cool to have 3 tiers: recent fast, middle regular, and old slow.  Maybe even more.  To this end I suggest a configuration something like (but we could probably come up with something better):
volume.dates.tier.1.volumes
volume.dates.tier.2.volumes
volume.dates.tier.2.days.back
volume.dates.tier.3.volumes
volume.dates.tier.3.days.back

Reply:
How about the following:
volume.tiers="name1,name2,name3"
volume.dates.default.volumes
volume.dates.name1.days.back
volume.dates.name1.volumes
volume.dates.name2.days.back
volume.dates.name2.volumes
volume.dates.name3.days.back
volume.dates.name3.volumes

Reply:
Resolved in newest commit